swan-cern/README.md

```diff
@@ -203,6 +203,12 @@ rmmod nouveau
 kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/1.0.0-beta4/nvidia-device-plugin.yml
 ```
 
+Provide custom rules for Falco deployment 
```

Comment:
Can you make this a section at the same level of e.g. Monitoring, this means with `###`.